CustomQuickchat/Components/Components/WebsocketManager.hpp

```diff
@@ -29,7 +29,7 @@ class WebsocketClientManager
 	WebsocketClientManager() = delete;
 	//~WebsocketClientManager() { StopClient(); };
 
-	std::shared_ptr<bool> connecting_to_server;
+	std::atomic<bool> connecting_to_server{false}
```

Comment:
This is missing a semicolon, fixed in the next commit. But just in case we want to build this commit separately or roll back the other one